### DISCUSSION TITLE
okk

### DISCUSSION THREAD

**Discussion by khanshorif331** [RiyaadHossain/Hoteltastic-Client#90] [Q&A] [answered]
Noo tension??Start programming???

**Comment by RiyaadHossain** [comment]
pleaseeeeeeeeeeeeeeeeeeee

**Comment by RiyaadHossain** [comment]
helloooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

**Comment by NishatFariza** [comment, marked as answer]
Okay, Thank you so much!! Yes I want to start my programming
